pyensembl/shell.py

```diff
@@ -55,72 +55,81 @@
 logger = logging.getLogger(__name__)
 
 
-def run():
-    parser = argparse.ArgumentParser(usage=__doc__)
-    parser.add_argument(
-        "--overwrite",
-        default=False,
-        action="store_true",
-        help="Force download and indexing even if files already exist locally")
-
-    root_group = parser.add_mutually_exclusive_group()
-
-    release_group = root_group.add_argument_group()
-    release_group.add_argument(
-        "--release",
-        type=int,
-        nargs="+",
-        default=[],
-        help="Ensembl release version, multiple releases may be specified.")
-
-    release_group.add_argument(
-        "--species",
-        default="human",
-        help="Which species to download Ensembl data for (default=%(default)s.")
-
-    path_group = root_group.add_argument_group()
-    path_group.add_argument(
-        "--reference-name",
-        type=str,
-        default=None,
-        help="Name of the reference, e.g. GRCh38")
-    path_group.add_argument(
-        "--annotation-name",
-        default=None,
-        help="Name of annotation source (e.g. refseq)")
-    path_group.add_argument(
-        "--annotation-version",
-        default=None,
-        help="Version of annotation database")
-    path_group.add_argument(
-        "--gtf",
-        type=str,
-        default=None,
-        help="URL or local path to a GTF file containing annotations.")
-    path_group.add_argument(
-        "--transcript-fasta",
-        type=str,
-        action='append',
-        default=None,
-        help="URL or local path to a FASTA files containing the transcript "
-        "data. This option can be specified multiple times for multiple "
-        "FASTA files.")
-    path_group.add_argument(
-        "--protein-fasta",
-        type=str,
-        default=None,
-        help="URL or local path to a FASTA file containing protein data.")
-
-    parser.add_argument("action",
-        type=lambda arg: arg.lower().strip(),
-        choices=("install", "delete-all-files", "delete-index-files"),
-        help="\"install\" will download and index any data that is  not "
+parser = argparse.ArgumentParser(usage=__doc__)
```

Comment:
this file can be reverted, since it's only related to the delete-source-files addition